src/joyride/sci.cljs

```diff
@@ -99,6 +100,10 @@
 
 (def core-namespace (sci/create-ns 'clojure.core nil))
 
+(def cljreplns (sci/create-ns 'clojure.repl))
+(def clj-repl-namespace (merge (sci/copy-ns clojure.repl cljreplns)
```

Comment:
I wouldn't do this. This copies the `clojure.repl` namespace from the host environment. SCI comes with its own `clojure.repl` namespace already. Just add to the config: `{:namespaces {'clojure.repl {'pst ...}}}`.

Reply:
Now fixed.